### PR TITLE
Introduced RsStatus.ByCode

### PR DESCRIPTION
**Pull request by olenagerasimova**
I've introduced class `RsStatus.ByCode` to obtain `RsStatus` instance by code.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#178](https://github.com/artipie/http/issues/178) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/72789339-0c0e-45ed-871d-929e55f72a7a); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/main/java/com/artipie/http/rs/RsStatus.java [hunk 1]
```diff
@@ -160,4 +163,42 @@
...
+        /**
+         * Status code.
+         */
+        private final String code;
+
+        /**
+         * Ctor.
+         * @param code Code
+         */
+        public ByCode(@Nullable final String code) {
```
Comment (Vatavuk) [inline src/main/java/com/artipie/http/rs/RsStatus.java:182]: @olenagerasimova we can add ctor that accepts int value to make this class more reusable

File: src/main/java/com/artipie/http/rs/RsStatus.java [hunk 2]
```diff
@@ -160,4 +163,42 @@
     public String code() {
         return this.string;
     }
+
+    /**
+     * RsStatus by code.
```
Comment (Vatavuk) [inline src/main/java/com/artipie/http/rs/RsStatus.java:168]: @olenagerasimova just expand the description a bit.

**Comment by olenagerasimova** [comment]
@Vatavuk thanks, corrected

**Comment by 0crat** [comment]
Job was finished in 29 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@Vatavuk please make sure you will find at least 3 issues during next CR, thanks

**Comment by Vatavuk** [comment]
@sereshqua ok

**Comment by sereshqua** [comment]
@0crat quality acceptable

**Pull request merged**